app/lib/create-reference.js

```diff
@@ -1,9 +1,67 @@
 /**
- * Generate unique reference number
- * ex. VV22-B471-F25C
- * @returns string
+ * Generate prefix according to the type of claim the reference being created for
```

Comment:
Feel like we are suddenly going full in on comments. Think we should try and follow the Defra coding standards guidance here and really keep them to an absolute minimum. I think the function below is very clear and self explanatory. Not sure it's needs a verbose explanation as well.

https://defra.github.io/software-development-standards/standards/common_coding_standards/

Reply:
I didnt feel like JSDocs fit in the same category as the comments bit mentioned in the coding standards - maybe I'm wrong for thinking that though. I felt like it was nice to ensure I was passing the correct arguments to some of the functions, using the IDE to do a bit of TS-esque stuff for me.

For this function, I agree its simple and we know what it does - but the arguments that are given to it are extremely specific; it felt like a good thing to help make sure the Dev passes the right strings?

Reply:
will remove for now - can maybe add back in?

Reply:
Thanks, agree it's maybe one for wider discussion before we make a final call.